prover/prover_fri_types/src/lib.rs

```diff
@@ -121,6 +118,11 @@ fn get_round_for_recursive_circuit_type(circuit_type: u8) -> AggregationRound {
         circuit_type if circuit_type == ZkSyncRecursionLayerStorageType::NodeLayerCircuit as u8 => {
             AggregationRound::NodeAggregation
         }
+        circuit_type
```

Comment:
I don't really like how this match looks, can we use something like ZkSyncRecursionLayerStorageType::from(circuit_type) and match options but not comparing u8?